Brokerages/Binance/BinanceBrokerage.Messaging.cs

```diff
@@ -121,7 +121,7 @@ private void OnMessageImpl(WebSocketMessage e)
                 {
                     var quote = objData.ToObject<BestBidAskQuote>();
                     EmitQuoteTick(
-                        _symbolMapper.GetLeanSymbol(quote.Symbol),
+                        _symbolMapper.GetLeanSymbol(quote.Symbol, SecurityType.Crypto, Market.Binance),
```

Comment:
Interesting we weren't using the mapper when subscribing to map lean symbols for broker symbols

Reply:
Yeah, it would have worked anyway without mapper because Lean tickers are the same as the Binance tickers